packages/user-operation-controller/src/UserOperationController.ts

```diff
@@ -94,6 +112,27 @@ export type UserOperationControllerOptions = {
   state?: Partial<UserOperationControllerState>;
 };
 
+export type AddUserOperationRequest = {
```

Comment:
Nit: What about moving some of the types here to `types.ts`?

Reply:
Great question!

My criteria for the `types` file are those that aren't tied to a specific file or exist beyond the package whether we define them or not.

In the case of the request and response objects, they are defined specifically for our public methods hence why I've left them in `UserOperationController.ts` for clarity.